### DISCUSSION TITLE
will mediation implemented to v1.x in the future?

### DISCUSSION THREAD

**Discussion by di-kang** [Poing-Studios/godot-admob-editor#93] [Q&A] [answered]
i'm using v1.3.4 on godot v3.5.2 and migrating current project to godot 4 is nearly impossible, at least for now it seems. 

so it would be great if mediation is planned to be implemented to v1.x in the near future.

**Comment by gumaciel** [comment, marked as answer]
We regret to inform you that we won't be able to implement mediation for v1, as we currently lack contributors for this endeavor. It is advisable to focus our efforts on supporting new versions of Godot instead. Sorry.

Check this: 
https://github.com/Poing-Studios/godot-admob-plugin/discussions/87
https://github.com/Poing-Studios/godot-admob-plugin/discussions/88

**Reply by di-kang** [reply]
no worries. i understand. thanks for the quick reply :)
